paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -66,6 +68,9 @@ internal class PaymentSheetPlaygroundViewModel(
     val customerSheetState = MutableStateFlow<CustomerSheetState?>(null)
     val customerAdapter = MutableStateFlow<CustomerAdapter?>(null)
 
+    @OptIn(ExperimentalCvcRecollectionApi::class)
+    val cvcCallback = CvcRecollectionEnabledCallback { true }
```

Comment:
We should add this as a playground setting, so we can test both states.

Reply:
Done 👍